.github/actions/setup/action.yml

```diff
@@ -7,12 +7,7 @@ runs:
       uses: actions/setup-node@v3
       with:
         node-version: 18.16.0
-    - name: Cache
-      uses: actions/cache@v3
-      id: cache
-      with:
-        path: '**/node_modules'
-        key: yarn-v1-${{ runner.os }}-${{ hashFiles('.nvmrc') }}-${{ hashFiles('**/yarn.lock') }}
+        cache: yarn
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏The default cache mechanism does not take into account `.nvmrc` file. That's why we created a custom cache key here.

With the proposed change the cache will not be invalidated when we bump the node version in `.nvmrc`

Reply:
Oh I see, I'll revert this change